apps/react-18-tests-v9/cypress.config.ts

```diff
@@ -1,3 +1,3 @@
-import baseConfig from '@fluentui/scripts/cypress/cypress.config';
+import { baseConfig } from '@fluentui/scripts/cypress';
```

Comment:
QQ: Is there anything that will prevent us from using `import baseConfig from '@fluentui/scripts/cypress/cypress.config';`?

Reply:
all these sub-folders will be packaged as last step (including export maps as well ) - which should throw error on such import. we wont do type rolluping for internal packages as that doesn't make much sense. we can enable lint rule to ban deep imports in those.